align_system/algorithms/outlines_regression_adm_comparative.py

```diff
@@ -300,6 +300,19 @@ def top_level_choose_action(self,
                                                         kdma_score_examples,
                                                         incontext_settings=kwargs.get("incontext", {}))
 
+        # Log true kdma values if present
+        true_kdma_values = {}
+        for choice_idx in range(len(available_actions)):
+            true_kdma_values[choices[choice_idx]] = available_actions[choice_idx].kdma_association
```

Comment:
Nit: Double checking that this doesn't cause issues for eval mode

Reply:
I checked, it doesn't cause an issue, the true values are just logged as null. In the MSE calculation script, I made it so MSE is only calculated when the true values are available and not null.